lib/logstash/outputs/opensearch/http_client/manticore_adapter.rb

```diff
@@ -76,13 +79,34 @@ def aws_iam_auth_initialization(options)
       @credentials = Aws::CredentialProviderChain.new(credential_config).resolve
     end
 
+    def basic_auth_initialization(options)
+      set_user(options)
+      set_password(options)
+    end
+
     def set_aws_region(region)
       @region = region
     end
 
     def get_aws_region()
       @region
     end
+
+    def get_user()
+      @user
+    end
+
+    def set_user(options)
+      @user = options[:auth_type]["user"]
```

Comment:
What happens if, for whatever reason, "user" is not present?

Reply:
It will throw an exception like ":exception=>java.lang.IllegalArgumentException: Username may not be null"